alvr/client_core/src/graphics/staging.rs

```diff
@@ -62,14 +62,24 @@ impl StagingRenderer {
         context: Rc<GraphicsContext>,
         staging_textures: [gl::Texture; 2],
         view_resolution: UVec2,
+        fix_limited_range: bool,
     ) -> Self {
         let gl = &context.gl_context;
         context.make_current();
 
+        // Add #defines into the shader after the first line
+        let mut frag_lines: Vec<&str> = include_str!("../../resources/staging_fragment.glsl")
+            .lines()
+            .collect();
+        if fix_limited_range {
+            frag_lines.insert(1, "#line 0 1\n#define FIX_LIMITED_RANGE");
+        }
+        let frag_str = frag_lines.join("\n");
```

Comment:
This is kinda obscure but doesn't matter too much, we are going to remove the option for v21 and force full range